dcos/marathon.py

```diff
@@ -388,6 +388,35 @@ def scale_app(self, app_id, instances, force=None):
         deployment = response.json()['deploymentId']
         return deployment
 
+    def scale_group(self, group_id, scale_factor, force=None):
+        """Scales a group with the requested scale-factor.
+        :param group_id: the ID of the group to scale
+        :type group_id: str
+        :param scale_factor: the requested value of scale-factor
+        :type scale_factor: int
```

Comment:
The type for `scale_factor` should be `float` and assume so in the code below. See my comment below.